### PR TITLE
Clean up Varnish files

### PR DESCRIPTION
**Pull request by pyrmont**
The Varnish lexer was written in a style inconsistent with that used in the current version Rouge. This PR fixes those problems.

### CODE REVIEW

File: lib/rouge/lexers/varnish.rb [hunk 1]
```diff
@@ -1,224 +1,130 @@
...
+        ]
+      end
+
+      # This is never used
+      # def self.routines
+      #   @routines ||= Set.new %w[
+      #     backend_error backend_fetch backend_response purge deliver fini hash
+      #     hit init miss pass pipe recv synth
+      #   ]
+      # end
```
Comment (pyrmont) [inline lib/rouge/lexers/varnish.rb]: @gfx These keywords are never used in the lexer (they were originally assigned to the `ROUTINES` constant). Do you know if they should be deleted or what token to use with them?

**Comment by pyrmont** [comment]
@blckct Thanks for submitting #1434! There are a bunch of changes that need to be made to the Varnish lexer to bring it into conformity with the way we try to write lexers now and so I created this PR to do that. I've added your keywords here.

Since I'm guessing you have some familiarity with Varnish, could you have a look at the code in this PR and let me know what you think?

File: lib/rouge/lexers/varnish.rb (hunk 1 above)
Comment (gfx) [reply lib/rouge/lexers/varnish.rb]: I see. Thank youl. Could you leave it as is in this PR? I'll make another PR to refine this lexer.

Comment (pyrmont) [reply lib/rouge/lexers/varnish.rb]: @gfx Will do!

File: lib/rouge/lexers/varnish.rb [hunk 2]
```diff
@@ -1,224 +1,131 @@
...
+
+      state :root do
+        # long strings ({" ... "})
+        rule %r/\{".*?"}/m, Str::Single
+
+        # comments
+        rule %r'/\*.*?\*/'m, Comment::Multiline
+        rule %r'(?:/|#).*', Comment::Single
```
Comment (julp) [inline lib/rouge/lexers/varnish.rb]: Shouldn't it be `%r'(?://|#).*'`?

Comment (pyrmont) [reply lib/rouge/lexers/varnish.rb]: @julp Thank you! Yes, it should be!

**Pull request merged**